yaksh/models.py

```diff
@@ -68,43 +68,6 @@
                 ('completed', 'Completed'),
               )
 
-instructions_data = dedent("""\
```

Comment:
This variable is used in create_demo_quiz method for instructions in quiz models.
Please check if that works for creating demo course and demo quiz.